Makefile

```diff
@@ -21,82 +21,114 @@ endif
 
 all: manager
 
-# Create a local docker registry, start kind cluster, and install Orkestra
+## Create kind cluster with local registry and prepopulate the local registry.
+setup-dev-env: kind-create prepopulate-kind-registry
+
+## Deploy the Orkestra helm chart (values-ci.yaml) with Orkestra controller disabled.
 dev:
-	make kind-create
 	helm upgrade --install orkestra chart/orkestra --wait --atomic -n orkestra --create-namespace --values ${CI_VALUES}
 
+## Deploy the Orkestra helm chart (values-ci.yaml) with Orkestra controller enabled (in debug mode).
 debug: dev
 	go run main.go --debug --log-level ${DEBUG_LEVEL}
 
+## Cleanup any previous Orkestra helm chart installation.
+clean:
```

Comment:
`clean` and `clean-all` is confusing. `clean` should clean everything as this is the understood behavior of `clean`, more granular cleaning can be done with things like `clean-chart` or `clean-cluster`